go.mod

```diff
@@ -181,6 +181,7 @@ require (
 	github.com/juju/gojsonpointer v0.0.0-20150204194629-afe8b77aa08f // indirect
 	github.com/juju/gojsonreference v0.0.0-20150204194633-f0d24ac5ee33 // indirect
 	github.com/juju/lru v0.0.0-20190314140547-92a0afabdc41 // indirect
+	github.com/juju/mgo/v3 v3.0.3 // indirect
```

Comment:
How did this sneak in here? Do we want it?

Reply:
its not actually used